renderdoc/replay/entry_points.cpp

```diff
@@ -573,7 +573,23 @@ void extractDeviceIDAndIndex(const string &hostname, int &index, string &deviceI
 
   deviceID = c;
 }
-string adbExecCommand(const string &device, const string &args)
+Process::ProcessResult execCommand(const string &cmd, const string &workDir = ".")
+{
+  RDCLOG("COMMAND: %s", cmd.c_str());
+
+  size_t firstSpace = cmd.find(" ");
+  string exe = cmd.substr(0, firstSpace);
+  string args = cmd.substr(firstSpace + 1, cmd.length());
+
+  Process::ProcessResult result;
+  Process::LaunchProcess(exe.c_str(), workDir.c_str(), args.c_str(), &result);
+  if(result.strStdout.length())
+    RDCLOG("STDOUT:\n%s", result.strStdout.c_str());
+  if(result.strStderror.length())
+    RDCLOG("STDERR:\n%s", result.strStderror.c_str());
```

Comment:
Yeh it's absolutely a useful debug feature, but I just think the value proposition is starting to swing around. Especially since hopefully this code should all be stable soon and there shouldn't be a need for people to check for the command output. There will be a happy balance somewhere I just think we'll have to change this default to be more judicious.